templates/autoscale-existing-vpc.template

```diff
@@ -0,0 +1,1049 @@
+            "Description": "(Optional) The destination path for health checks. This path must begin with a '/' character, and can be at most 1024 characters in length."
+        },
+        "InternalLoadBalancerDNSName": {
+            "Type": "String",
+            "Default": "",
+            "Description": "(Optional) The DNS name of an existing internal load balancer used to route traffic from a FortiGate to targets in a specified target group. Leave it blank if you don't use an existing load balancer."
+        },
+        "GetLicenseGracePeriod": {
+            "Type": "Number",
+            "Default": 600,
+            "MinValue": 300,
+            "ConstraintDescription": "must be a valid number not less than 300.",
+            "Description": "The minimum time (in seconds) permitted before a distributed license can be revoked from a non-responsive FortiGate and re-distributed. Minimum is 300."
+        },
+        "FortiAnalyzerIntegrationOptions": {
+            "Type": "String",
+            "Default": "yes",
+            "AllowedValues": ["yes", "no"],
+            "ConstraintDescription": "must choose from the provided options.",
+            "Description": "Choose 'yes' to incorporate FortiAnalyzer into Fortinet FortiGate Auto Scaling to use extended features that include storing logs into FortiAnalyzer."
+        },
+        "FortiAnalyzerInstanceType": {
+            "Type": "String",
+            "Default": "m4.large",
+            "AllowedValues": [
+                "c4.large",
+                "c4.xlarge",
+                "c4.2xlarge",
+                "c4.4xlarge",
+                "c4.8xlarge",
+                "c5.large",
+                "c5.xlarge",
+                "c5.2xlarge",
+                "c5.4xlarge",
+                "c5.9xlarge",
+                "c5.12xlarge",
+                "c5.18xlarge",
+                "c5.24xlarge",
+                "d2.xlarge",
+                "d2.2xlarge",
+                "d2.4xlarge",
+                "d2.8xlarge",
+                "h1.2xlarge",
+                "h1.4xlarge",
+                "h1.8xlarge",
+                "h1.16xlarge",
+                "m4.large",
+                "m4.xlarge",
+                "m4.2xlarge",
+                "m4.4xlarge",
+                "m4.10xlarge",
+                "m4.16xlarge",
+                "m5.large",
+                "m5.xlarge",
+                "m5.2xlarge",
+                "m5.4xlarge",
+                "m5.8xlarge",
+                "m5.12xlarge",
+                "m5.16xlarge",
+                "m5.24xlarge"
+            ],
+            "ConstraintDescription": "must be a valid EC2 instance type.",
+            "Description": "The instance type to launch as FortiAnalyzer On-Demand instances. There are t2.small and compute-optimized instances such as m4 and c4 available with different vCPU sizes and bandwidths. For more information about instance types, see https://aws.amazon.com/ec2/instance-types/"
```

Comment:
Description mentions t2.small instances. AllowedValues does not list any t2.small instances.

Reply:
> The description mentions t2.small instances. AllowedValues does not list any t2.small instances.

@klc178 I checked that FAZ doesn't offer t2 series so would you please update the description?